Grid/Source/Entity.php

```diff
@@ -437,7 +441,7 @@ public function execute($columns, $page = 0, $limit = 0, $maxResults = null, $gr
 
                     $columnForFilter = (!$column instanceof JoinColumn) ? $column : $columnsById[$filter->getColumnName()];
 
-                    $fieldName = $this->getFieldName($columnForFilter, false);
+                    $fieldName = $this->getFieldName($columnForFilter, false, $hasHavingClause);
```

Comment:
if you add a  parameter to this function, you should change its signature wich is actually

```
protected function getFieldName($column, $withAlias = false)
```